src/pages/resultsView/cancerSummary/CancerSummaryContainer.tsx

```diff
@@ -79,21 +79,27 @@ export default class CancerSummaryContainer extends React.Component<
         }
     }
 
-    // this is used to map study id to study shortname
-    private mapStudyIdToShortName(str: string) {
+    // this is used to map study id to study name
+    private mapStudyIdToName(str: string) {
         if (str in this.props.store.physicalStudySet) {
-            return this.props.store.physicalStudySet[str].shortName;
+            const studyname = this.props.store.physicalStudySet[str].name;
+            if (studyname.indexOf('(') > 50) {
```

Comment:
since you need ```studyname.indexOf('(') > 50``` again, first assign to local var and then follow with if statement